portfolio_answer.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    
+    <link rel="stylesheet" href="style.css" />
+    
+
+</head>
+<body>
+  
+    <style scoped>
+        
+        /* 0 to 299 */
+        .on-the-fly-behavior {
+            background-image: url('particular_ad_small.png'); 
+        }
+        /* 300 to X */
+        @media (min-width: 300px) { /* or 301 if you want really the same as previously.  */
+            .on-the-fly-behavior {   
+                background-image: url('particular_ad.png');
+            }
+        }
+        </style>
```

Comment:
Please use external CSS, kindly add CSS related code to a CSS file. Adding CSS rules to each HTML element takes time and makes your HTML structure unorganized.